README.md

```diff
@@ -233,7 +233,8 @@ sveld({
   },
   json: true,
   jsonOptions: {
-    outFile: "docs/src/COMPONENT_API.json",
+    outFile: "docs/src/mycomponentname-api.json",
+    outDir: "docs/src/mycomponentname-api.json", // optional, if specified, all JSON files will be written in this folder
```

Comment:
Having both the `outFile` and `outDir` is misleading. When I read this, I would expect a single file and individual files to be generated.

Furthermore, `outDir` shouldn't be a file but a folder.

I've updated the README: https://github.com/carbon-design-system/sveld/pull/78/commits/173b7512d03c61ec07fc1a90856a4a078d6ff055